tools/scripts/tests.toml

```diff
@@ -4,6 +4,14 @@
 
 # This is a cargo-make file included in the toplevel Makefile.toml
 
+[tasks.check-all-features]
+description = "Build all permutations of all features"
+category = "ICU4X Development"
+install_crate = { crate_name = "cargo-all-features", binary = "cargo-check-all-features", test_arg = ["--help"] }
+install_crate_args = ["--version", "^1.4"]
```

Comment:
Nit: If we require 1.7 we should update it here I think

Reply:
Argh, missed this, will make a PR